Compile server/test/* with -lm -lpthread option, or compile will fail--- a/server/tests/Makefile.am
+++ b/server/tests/Makefile.am
@@ -17,6 +17,7 @@
 LDADD =								\
 	$(top_builddir)/spice-common/common/libspice-common.la	\
 	$(top_builddir)/server/libspice-server.la		\
+	$(SPICE_NONPKGCONFIG_LIBS)		\
 	$(NULL)
 
 COMMON_BASE =					\
